src/ts/navigation/navigation.test.ts

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
I've kept this around as a basic initialisation test but honestly given the cypress tests cover everything anyway and are much better suited I'm inclined to delete this completely? How do folks feel about that? It's a pretty fake test as it stands.

Reply:
I think getting rid of it is fine